src/client/src/orangehrmRecruitmentPlugin/components/RecruitmentStatus.vue

```diff
@@ -165,7 +165,7 @@ export default {
   },
   beforeMount() {
     this.isLoading = true;
-    this.status = this.candidate.status;
+    this.status = this.candidate.status; // TODO
```

Comment:
saw this is couple places. this status is sent from BE right? anything pending in FE?

Reply:
yes. still checking refactored code is working properly